hack/generator/pkg/jsonast/jsonast.go

```diff
@@ -235,7 +235,17 @@ func enumHandler(ctx context.Context, scanner *SchemaScanner, schema *gojsonsche
 
 	var values []astmodel.EnumValue
 	for _, v := range schema.Enum {
-		id := scanner.idFactory.CreateIdentifier(v, astmodel.Exported)
+		vTrimmed := strings.Trim(v, "\"")
+
+		// Some specs include boolean enums with quotes around the boolean literals
+		if baseType == astmodel.BoolType {
+			v = vTrimmed
+		}
+
+		// TODO: This is a bit of a hack as we don't have a way to handle this generically right now
+		// TODO: for an arbitrary non-renderable character
+		// use vTrimmed as seed for identifier as it doesn't have quotes surrounding it
+		id := scanner.idFactory.CreateIdentifier(vTrimmed, astmodel.Exported)
```

Comment:
Odd that you need this - the `identifierFactory` includes a regex to strip out non-word characters:

```
var filterRegex = regexp.MustCompile(`[\W_]`)
```

We should probably simplify your change here by enhancing the regex; the goal of the factory is to hide away all that complexity.

Reply:
Removing the quotes here is to get the "replace" bit to be less awkward.

I could've said: `"\"*\""` is mapped to `star` but that seems icky too.

Reply:
I've dug into it a little and now I understand the problem; it is a bit icky. I'm happy enough with what you've got here, not need to change it.